openpathsampling/storage/storage.py

```diff
@@ -25,6 +25,8 @@ class Storage(NetCDFPlus):
     simulation. This allows effective storage of shooting trajectories
     """
 
+    USE_FEATURE_SNAPSHOTS = False
```

Comment:
I tested both cases but I have not yet implemented a test that compares both cases. Will do that once we agree that the overall structure is reasonable. I can also add a speed comparison now and see if the feature implementation is slower.

I realized though that the new ToySnapshot seems to be faster, but have not tested it. In my feeling the MSTIS examples are running faster now. This is because all the energy handling and creating of Configuration objects is not necessary anymore.